torch/onnx/__init__.py

```diff
@@ -192,6 +192,8 @@ def select_model_mode_for_export(model, mode):
     A context manager to temporarily set the training mode of 'model'
     to 'mode', resetting it when we exit the with-block.  A no-op if
     mode is None.
+
+    In verison 1.5 changed to this from set_training
```

Comment:
minor: typo in version here. also `set_training` looks like it needs double backticks

Reply:
ok, should be 1.6

Reply:
and "version" not "verison"